### PR TITLE
🚀 Release `1.3.2`

### PR DESCRIPTION
**Pull request by thesan**
### Fixed

- Incorrect values in the "My Roles" tab.
- Working group history tab crashing.
- Infinite auto scroll to a post after opening a link to this post.
- Unclear councilor reward widget.
- Some posts overflowing on Webkit browsers.
- Improve stake and constitutionality visibility when creating a proposal.
- Copy improvements: proposal discussion status and voting rounds tabs tooltips, and forum empty category.

### Removed
- Buggy link icon on proposal lists.

### CODE REVIEW

File: packages/ui/src/working-groups/components/Roles/RolesList.tsx [hunk 1]
```diff
@@ -80,13 +97,17 @@ const RolesListItem = ({ worker }: { worker: Worker }) => {
...
           <ToggleableSubscriptionWide>Earned total</ToggleableSubscriptionWide>
         </OpenItemSummaryColumn>
         <OpenItemSummaryColumn>
           <TextInlineBig>
-            <TokenValue value={earnings} />
+            <Tooltip
+              tooltipText={<NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>}
+            >
+              <NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>
+            </Tooltip>
```
Comment (traumschule) [inline packages/ui/src/working-groups/components/Roles/RolesList.tsx:110]: This not very DRY. No need for the tooltip if it shows exactly the same:
```suggestion
              <NextPaymentValue>{payout?.gte(BN_ZERO) ? payout?.toString() : '–'}</NextPaymentValue>
```

**Review by traumschule — APPROVED** [review]
:rocket:

**Review by dmtrjsg — APPROVED** [review]
Lezzgo 🚀

File: packages/ui/src/working-groups/components/Roles/RolesList.tsx (hunk 1 above)
Comment (thesan) [reply packages/ui/src/working-groups/components/Roles/RolesList.tsx:110]: Nice catch I created an issue for this: #4323

**Pull request merged**